### PR TITLE
minor updates

### PR DESCRIPTION
**Pull request by brekk**

### CODE REVIEW

File: pkg/update-readme [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+cp ../README.md .
```
Comment (aboeglin) [inline pkg/update-readme:3]: I'd just reverse that and move it to the scripts folder, what do you think?

Then it'd become that:
```suggestion
cp ./README.md ./pkg/
```

And potentially we want to run that script directly from the github workflow for releases as well to be sure we're in sync.

**Review by aboeglin — CHANGES_REQUESTED** [review]
One small comment about the update-readme script but overall very good stuff!

File: pkg/update-readme (hunk 1 above)
Comment (brekk) [reply pkg/update-readme:3]: Sounds good to me.

**Comment by aboeglin** [comment]
If we can wrap that up before Sunday we can target the PR to the `v0.8.0` branch directly and have it released with the weekly release. Otherwise it goes in `v0.9.0`. No pressure though, I don't see it as being urgent but definitely nice to have.

**Comment by brekk** [comment]
Also gonna close this one as it's very out of date

**Pull request closed without merging**